client/src/pages/VoteResults/VoteResults.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import "./styles.css";
+
+export type UserRecord = {
+    name: string;
+    answer: "yes" | "no" | "if-needed";
+};
+
+export type DateRecord = {
+    timestamp: string;
+    record: UserRecord[];
```

Comment:
nazev by mel byt `records` (mnozne cislo) - lepe to reflektuje, ze se jedna o pole